config/environments/production.rb

```diff
@@ -74,16 +74,18 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
- config.action_mailer.default_url_options = { host: 'http://www.meucontroledeponto.com.br' }
-
- ActionMailer::Base.smtp_settings = {
-   address: 'smtp.sendgrid.net',
-   port: '587',
-   authentication: :plain,
-   enable_starttls_auto: true,
-   user_name: ENV['SENDGRID_USER'],
-   password: ENV['SENDGRID_PASS'],
-   domain: 'meucontroledeponto.com.br'
- }
+  config.action_mailer.default_url_options = { host: 'http://www.meucontroledeponto.com.br' }
+
+  config.lograge.enabled = true
 
 end
+
+ActionMailer::Base.smtp_settings = {
+  address: 'smtp.sendgrid.net',
+  port: '587',
+  authentication: :plain,
+  enable_starttls_auto: true,
+  user_name: ENV['SENDGRID_USER'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.